### ISSUE TITLE
Provide `parserServices` when using `@typescript-eslint/parser`

### ISSUE THREAD

**Issue by Yash-Singh1** [fkling/astexplorer#665]
**Describe the bug**

When I am trying to use the `parserServices` to make a typed rule in typescript eslint, the `parserServices` is undefined

**To Reproduce**
1. Go to the ast explorer
2. Select the parser as typescript eslint
3. Try to access parser services
4. Notice it is not filled

**Expected behavior**

parserServices are provided

**Screenshots**

**Browser (please complete the following information):**
 - OS: Ubuntu 20
 - Browser chrome

**astexplorer settings:**
- Selected parser: typescript eslint
- Selected transformer (if applicable): typescript eslint
- Contents of the local storage key `explorerSettingsV1` (code can be removed if you don't want it to be public)

**Additional context**

The problem is related to setting parseroptions.object, as seen here

https://github.com/typescript-eslint/typescript-eslint/blob/830b9463fc80ee3deba435f273e73fa6339564e7/packages/utils/src/eslint-utils/getParserServices.ts#L5

**Comment by DerekRies** [comment]
I don't think this is a bug, but more of a feature request to include an actual `@typescript-eslint` transform. Afaik there's only the `@typescript-eslint/parser` and an ESLint transform. The `parserServices` stuff is included with `@typescript-eslint/utils`, not the parser, nor the base eslint package.

Adding that would make this an exceptionally powerful tool though.
